### PR TITLE
Got the rerouting change to follow the flow we discuss and also got t…

### PR DESCRIPTION
**Pull request by PhatherJean**
…he state to follow through to the end of the flow will need to remove the console calls

### CODE REVIEW

**Review by roberttierie — APPROVED** [review]
Everything looks great. Super job. But we need to remove some console.logs for production later on.

File: src/components/pages/ActivityCheckIn/ActivityCheckIn.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ import styled from 'styled-components';
 // import axios from 'axios';
 
 const CheckEventActivityEmoji = props => {
-  const studentId = props.location.state;
+  console.log(props);
```
Comment (Ironipad64) [inline src/components/pages/ActivityCheckIn/ActivityCheckIn.js:9]: dont forget to take these out

File: src/components/pages/EmojiCheckIn/EmojiContainerCheckIn.js [hunk 2]
```diff
@@ -4,37 +4,30 @@ import NavBar from '../../common/NavBar';
...
 import axios from 'axios';
 
-const EmojiContainerCheckIn = props => {
-  const studentInfo = {
-    studentId: props.location.state,
-    reaction: '',
-  };
-  const [formState, setFormState] = useState({ studentInfo });
+const EmojiContainerCheckIn = ({ setStatus, status }) => {
+  console.log(status);
```
Comment (Ironipad64) [inline src/components/pages/EmojiCheckIn/EmojiContainerCheckIn.js:8]: and this one

File: src/components/pages/EmojiCheckOut/EmojiContainerCheckOut.js [hunk 3]
```diff
@@ -5,34 +5,29 @@ import styled from 'styled-components';
 import axios from 'axios';
 
 const EmojiContainerCheckIn = props => {
-  const studentInfo = {
-    studentId: props.location.state,
-    reaction: '',
-  };
-  const [formState, setFormState] = useState({ studentInfo });
+  console.log(props);
```
Comment (Ironipad64) [inline src/components/pages/EmojiCheckOut/EmojiContainerCheckOut.js:8]: !

**Review by Ironipad64 — APPROVED** [review]

**Pull request merged**